docs/QuickStart.md

```diff
@@ -0,0 +1,148 @@
+# QuickStart
+
+## Installation
+
+We support Linux and MacOS in current stage, Ubuntu 16.04 or higher, along with MacOS 10.14.1 are tested and supported. Simply run the following `pip install` in an environment that has `python >= 3.5`.
+
+```bash
+    python3 -m pip install --upgrade nni
+```
+
+Note:
+
+* `--user` can be added if you want to install NNI in your home directory, which does not require any special privileges.
+* If there is any error like `Segmentation fault`, please refer to [FAQ][1]
+* For the system requirements of NNI, please refer to [Install NNI][2]
+
+## "Hello World" example: MNIST
+
+As long as you provide data sets, training methods and search spaces, NNI can automatically generate training experiments, adjust hyper-parameters, and find the optimal hyper-parameter configuration within the given time or number of experiments.
+
+Here is a QuickStart example to run a NNI experiment on MNIST.
+
+**Three things required to do when using NNI**
+
+1. Give a `Search Space` file in json, includes the `name` and the `range` (discrete valued or continuous valued) of hyper-parameters you need to search. For example: [search_space.json][3]
```

Comment:
I think 'distribution' is better than 'range'

Reply:
ok, will change to distribution~